pallets/aggregate/src/data.rs

```diff
@@ -209,13 +296,16 @@ impl<
 {
     /// Create a new domain.
     ///
+    #[allow(clippy::too_many_arguments)]
     pub fn create(
         id: u32,
         owner: User<A>,
         next_aggregation_id: u64,
         max_aggregation_size: AggregationSize,
         publish_queue_size: u32,
+        aggregate_rules: AggregateSecurityRules,
         ticket: Option<Ticket>,
+        delivery: DeliveryParams<A, B>,
```

Comment:
Minor point here: if someone doesn't want to send aggregations anywhere he is still forced to construct this data structure

Reply:
Yes, and point to `None` a destination but can ask money if he want.

Reply:
To be more explicit: that's exactly the case where you're using attestation bot but you want to be payed. Anyway the owner is optional if the caller is not the manager.

If your're scared about the interface you should always remember that there is a library (`zkverifyjs`) between the user and the extrinsic, and this library can make this parameter optional and fill it with the default value: this is quite more flexible than put a `Option` here and implement the default value in the consensum.